packages/instrumentation-mysql/test/index.metrics.test.ts

```diff
@@ -93,20 +105,14 @@ describe('mysql@2.x-Metrics', () => {
   });
 
   describe('#Pool - metrics', () => {
-    let pool: mysqlTypes.Pool;
+    let pool: Pool;
 
     beforeEach(() => {
       initMeterProvider();
       instrumentation.disable();
       instrumentation.enable();
       inMemoryMetricsExporter.reset();
-      // credentials to connect to pool if you run docker locally using 'npm run docker:start' from 'examples/mysql' folder
```

Comment:
Reviewer note: The `docker:start` script was removed a long while back. This comment is obsolete.